tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -32,6 +32,9 @@ const notes = (place) => $(`[name="/data/${place}/notes"]`);
 const writeNamePlace = (place) => $(`[name="/data/${place}/is_name_generated"][value="false"]`);
 const contactCard = () => $('.card h2');
 const contactCardIcon = (name) => $(`.card .heading .resource-icon[title="medic-${name}"]`);
+const pregnancyLabel = () => $('.content-pane .meta > div > .card .action-header h3');
+const visitLabel = () => $('.content-pane .meta > div > .card .row label'); 
+const numberOfReports = () => $(('.content-pane .meta > div > .card .row p'));
```

Comment:
@lorerod @ngaruko these 3 selectors have the same beginning `.content-pane .meta > div > .card .row` that can be a constant.